docs/Guides/Recommendations.md

```diff
@@ -305,32 +305,32 @@ readinessProbe:
 <a id="capacity"></a>
 
 In order to rightsize the production environment for your Fastify application, 
-you are highly recommended to perform your own measurements against 
+it is highly recommended that you perform your own measurements against 
 different configurations of the environment, which may
 use real CPU cores, virtual CPU cores (vCPU), or even fractional
 vCPU cores. We will use the term vCPU throughout this
 recommendation to represent any CPU type.
 
-You can use such tools as [k6](https://github.com/grafana/k6) 
-or [autocannon](https://github.com/mcollina/autocannon) for conducting
-the necessary performance tests.
+Tools such as [k6](https://github.com/grafana/k6) 
+or [autocannon](https://github.com/mcollina/autocannon) can be used for
+conducting the necessary performance tests.
 
-That said, you may also consider the following as a rule of a thumb:
+That said, you may also consider the following as a rule of thumb:
```

Comment:
@Fdawgs 
Why not rule of two thumbs? Every human has two of them?

Reply:
🤦